src/components/Tabs/components/Tab/Tab.tsx

```diff
@@ -50,96 +53,75 @@ class Tab extends React.PureComponent<CombinedProps, never> {
     if (itemHadFocus && selected && panelID != null) {
       focusPanelID(panelID);
     }
-  }
 
-  componentDidUpdate(previousProps: TabProps) {
-    const {selected: wasSelected} = previousProps;
-    const {focused, measuring, selected, panelID} = this.props;
+    /* eslint-disable-next-line react-hooks/exhaustive-deps */
```

Comment:
It would be nice if we didn't have to toss in an ignore 😬 combining the effects might help this